styles.css

```diff
@@ -21,13 +36,28 @@
     position: absolute;
 }
 
+/* upper purple bar (block) specific coordinates */
 #menu::before {
     left: 0;
     top: -20px;
 }
 
+/* upper purple bar (block) specific coordinates */
 #menu::after {
     left: 0;
     top: 20px;
 }
 
+/* upper purple bar: connected selector-container to upper purple bar -setting it to transform(rotate) */
+#container:hover #menu::before {
+    top: 0;
+    transform: rotate(45deg);
+    transition: ease 1s;
+}
+
+/*lower purple bar: connected selector-container to lower purple bar -setting it to transform(rotate) */
+#container:hover #menu::after {
+    top: 0;
+    transform: rotate(-45deg);
+    transition: ease 1s;
+}
```

Comment:
Als je onderstaande code had gebruikt en de 2 transitions hierboven had weg gelaten had je een mooiere overgang gehad (zoals in het voorbeeld).

````
#menu,
#menu::before,
#menu::after {
    transition: all .2s ease-in-out;
}